src/guide/advanced/permissions.md

```diff
@@ -0,0 +1,286 @@
+# Permissions
+
+Accounts need permission tokens for various actions on a blockchain, e.g.
+to mint or burn assets.
+
+There is a difference between a public and a private blockchain in terms of
+permissions granted to users. In a public blockchain, most accounts have
+the same set of permissions. In a private blockchain, most accounts are
+assumed not to be able to do anything outside of their own account or
+domain unless explicitly granted said permission.
+
+Having a permission to do something means having a `PermissionToken` to do
+so. There are two ways for users to receive permission tokens:
+
+1. Permission tokens can be granted as part of one-time allowance.
+
+   In this case, when a user mints a permission token, they are granted a
+   fixed amount `X` of permission to do `Y`. Once the user performs the `Y`
+   operation `X` times, the token expires.
```

Comment:
You probably think so because of `CanTransferOnlyFixedNumberOfTimesPerPeriod`. This token initialy was a demostration for *Orillion* project. This token does not expire, it just sets the limit of transfers

Reply:
this is something that was already in the tutorial in the ISI chapter, I've taken this info from there. I've updated this now.